src/include/stir/listmode/CListModeDataROOT.h

```diff
@@ -51,6 +51,20 @@ START_NAMESPACE_STIR
   We currently support only ROOT output using the 'Cylindrical PET' and 'ECAT' systems
   from GATE.
 
+  The Scanner geometry is set in multiple places. In hroot there is the option to use the
+  Scanner name to pull a Scanner from the library. Another option is to use
+  Scanner::User_defined_scanner and set the members: num_rings, num_detectors_per_ring,
+  max_num_non_arccorrected_bins, inner_ring_diameter, average_depth_of_interaction,
+  ring_spacing and bin_size, manually.
+
+  In any case this scanner must checkout the geometry created by the repeaters section
+  (which will initialise the InputStreamFromROOTFile.
+  And of course, match the Scanner used by other parts of STIR (e.g. ProjDataInfo e.t.c).
+
+  Finally, in the case that Scanner::Unknown_scanner the algorithm will use the flag
+  give_it_a_try and will try to create the scanner from the members. Subject to concistency
```

Comment:
I wouldn't mention `give_it_a_try` in the class doc, as it's an internal variable, and the reader might think it's a flag she can set.